Pollo/Extensions/Utils.swift

```diff
@@ -138,7 +130,8 @@ func formatResults(results: [String: JSON]) -> [String: PollResult] {
     results.forEach { (key, value) in
         let text = value[ParserKeys.textKey].stringValue
         let count = value[ParserKeys.countKey].intValue
-        pollResults[key] = PollResult(text: text, count: count)
+        let letter = value[ParserKeys.letterKey].intValue
```

Comment:
instantiate constants in the same order as the passed arguments?

Reply:
Fixed